cluster/examples/kubernetes/ceph/pool.yaml

```diff
@@ -12,6 +12,8 @@ metadata:
 spec:
   # The failure domain will spread the replicas of the data across different failure zones
   failureDomain: host
+  # number for replica per failure domain, the value must be lower than the number of replica and a power of 2
+  # replicasPerFailureDomain: 2
```

Comment:
This setting only applies to replication, not EC. How about moving it under the `replicated` element?